### ISSUE TITLE
Loading state not announced by screen reader

### ISSUE THREAD

**Issue by rheydt** [JedWatson/react-select#3351]
When using the async version of react-select (I'm using v2), the loadingMessage (or other reference to the loading state) is not announced by the screen reader while search results are loading. Instead, the screen reader announces that you have "0 options for search term [x]" until the results load, which is misleading. 

This behavior can be seen in the examples on the Async docs page - [https://react-select.com/async](https://react-select.com/async). I tested using Voiceover on Chrome and Safari, but since the screen reader text is manually provided by react-select (in the A11yText component), this bug should be consistent across screen readers and browsers. 

A possible solution is to pass the loadingMessage text to the A11yText component when isLoading is true.

**Comment by elizabethsjudd** [comment]
We've recently had this pointed out during an accessibility audit. Is there an ETA on when it may be completed?
